app/src/main/java/com/example/moviedb/ui/base/BaseFragment.kt

```diff
@@ -14,8 +15,18 @@ abstract class BaseFragment<T : ViewDataBinding> : Fragment() {
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         mBinding = DataBindingUtil.inflate(LayoutInflater.from(activity), getLayoutResourceId(), container, false)
+        setupView()
+        loadData()
         return mBinding.root
     }
 
     abstract fun getLayoutResourceId(): Int
```

Comment:
nếu mà k cần yêu cầu tham số truyền vào thì dùng abstract thuộc tính 
```suggestion
    @get:LayoutRes
    abstract val layoutResourceId : Int
```
lúc override nhớ viết 
```suggestion
    val layoutResourceId: Int get() = R.layout.chim_se_di_nang
```